config/sources/s500.conf

```diff
@@ -13,18 +13,19 @@ GOVERNOR=interactive
 
 case $BOARD in
 	guitar)
-	UBOOT_FILES="$SRC/lib/bin/bootloader-guitar.bin u-boot-dtb.img"
+	BOOTLOADER_BIN="s500-bootloader-guitar.bin"
+	UBOOT_FILES="$SRC/lib/bin/$BOOTLOADER_BIN u-boot-dtb.img"
 	;;
 
 	roseapple)
-	UBOOT_FILES="$SRC/lib/bin/bootloader-roseapple.bin u-boot-dtb.img"
+	BOOTLOADER_BIN="s500-bootloader-roseapple.bin"
+	UBOOT_FILES="$SRC/lib/bin/$BOOTLOADER_BIN u-boot-dtb.img"
 	;;
 esac
 
 write_uboot_platform()
 {
-	[ -f "$1/s500-bootloader-guitar.bin" ] && dd if=$1/bootloader-guitar.bin of=$2 bs=512 seek=4097 conv=fsync > /dev/null 2>&1
-	[ -f "$1/s500-bootloader-roseapple.bin" ] && dd if=$1/bootloader-roseapple.bin of=$2 bs=512 seek=4097 conv=fsync > /dev/null 2>&1
+	dd if=$1/$BOOTLOADER_BIN of=$2 bs=512 seek=4097 conv=fsync > /dev/null 2>&1
```

Comment:
Without fix it's impossible to build an image.
it gives an error file missing "U-boot file not found"
But to make it short i revert then we see further.